Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
After how I was treated when I tried submitting a fix for a contrast 
problems?  If Wackerbarth still has say in matters, no thanks.


On 6/1/2016 12:35 AM, Scott Lahteine wrote:
>
> In Marlin/ultralcd.cpp 
> <https://github.com/MarlinFirmware/Marlin/pull/2105#discussion_r65314494>:
>
> > @@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
> >  #ifdef HAS_LCD_CONTRAST
> >    static void lcd_set_contrast() {
> >      if (encoderPosition != 0) {
> > -      lcd_contrast -= encoderPosition;
> > -      lcd_contrast &= 0x3F;
> > +      #ifdef U8GLIB_LM6059_AF
> > +        lcd_contrast += encoderPosition;
>
> @eboston <https://github.com/eboston> Finally got those REVERSE 
> options for the menu into the code. At the moment, trying to resolve 
> an issue related to LCD contrast on graphical displays. I wonder if 
> you wouldn't mind adding your 2¢ to the conversation, at #3912 
> <https://github.com/MarlinFirmware/Marlin/issues/3912>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/MarlinFirmware/Marlin/pull/2105/files/082ed3beedfaffaa7fb8b8e2ae4fa2f0f9f8aca4#r65314494>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AH3tderjmTlInSH4ZvnLPSbt7E3H6w71ks5qHTZfgaJpZM4Ed7jG>.
>